RDF.js

```diff
@@ -967,6 +967,16 @@ function importItem(newItem, node) {
 	
 	// type
 	var type = getFirstResults(node, [n.dc+"type", n.dc1_0+"type", n.dcterms+"type"], true);
+	
+	/**CUSTOM ITEM TYPE  -- Currently only Dataset **/
+	if (type && type.toLowerCase() == "dataset") {
+		if (newItem.extra) {
+			newItem.extra += "\ntype: dataset";
+		}
+		else newItem.extra = "type: dataset";
```

Comment:
again, I need to test this, but I'm pretty sure type is correct here, as this is used by citeproc and hence relies on CSL json (which uses type, not itemType).